doc/release/technotes/README.llvm.rst

```diff
@@ -33,6 +37,7 @@ generation, and also support --llvm-wide-opt:
   make # you might want to do e.g. make -j 16 for a parallel build
 
 Note:
+
 * If you have a built llvm in third-party/llvm/install, even if you
   forget to export CHPL_LLVM=llvm, the default will be to use the built llvm.
```

Comment:
Couple of other spots too. I'll let you get your next commit in, and see if I spot anymore after that.